time stamp
uint16[] id_list

float64[] position__deg
float64[] profile_vel__deg_s
float64[] profile_acc__deg_ss